java-2d-array

```diff
@@ -0,0 +1,44 @@
+import java.io.*;
+import java.math.*;
+import java.security.*;
+import java.text.*;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.regex.*;
+
+public class Solution {
+
+
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
what is the point of making Scanner static final? this makes the Scanner object's resource not releasable by the garbage collector.
I know this comes from the original example but it's a wrong habit. I my opinion every instance which uses some I/O resource should be created locally **(unless some other circumstances are not require other solution)**. That makes easier to handle it's resource usage.